Sources/LocalNetworkConnectivity/VLCNetworkLoginViewController.m

```diff
@@ -71,11 +71,6 @@ - (void)viewDidLoad
                                               initWithTitle:NSLocalizedString(@"BUTTON_CONNECT", nil)
                                               style:UIBarButtonItemStyleDone target:self
                                               action:@selector(connectLoginDataSource)];
-    if (@available(iOS 13.0, *)) {
```

Comment:
now navigationBar changes color without this code (cause of extension) - and without lag!